tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -79,8 +79,8 @@ static_assert(ranges::enable_borrowed_range<span<int>>);
 static_assert(ranges::enable_borrowed_range<span<int, 3>>);
 #endif // __cpp_lib_concepts
 
-// We provide non-standard guarantees that span and its iterator types are trivially copyable;
-// P2251 may eventually standardize that guarantee for span itself.
+// Since P2251R1 span is guaranteed to be trivially copyable;
+// We provide non-standard guarantees that also its iterator types are trivially copyable.

```

Comment:
We don't typically comment on when and why behavior is required. I'd rather see the `span` tests here without comment, and the `span::iterator` tests interleaved into 95-102 with ", and that iterators are trivially copyable" appended to the comment on 93-94.

Reply:
I've followed your suggestions although I think a Standard citation is worthwhile (as it clearly draws a distinction between the Standard and non-Standard parts).